data/campaigns/Descent_Into_Darkness/scenarios/12_Endless_Night.cfg

```diff
@@ -10,68 +10,103 @@
+                    story= _ "<i>“As for me, I will make my journal conspicuous on this battlefield. I suspect that most will fear it enough to try to destroy it, but I have placed several wards on my journal to protect it. This way, it should survive long enough to make its way into the hands of someone more open-minded. Tath, in particular, is home to many mages. It only takes one who is willing to learn.”</i>"
                     {STORYTXT_BACKGROUND book.jpg}
                 [/part]
                 [part]
-                    story= _ "That final entry in the book gnaws at Mal Keshar. While he was being used by Darken Volk as a pawn, both of them were being using as pawns by the author of the book itself."
+                    story= _ "The final entry gnaws at Mal Keshar. While he had been Darken Volk’s pawn, both of them were being using as pawns by the author of the book. Has his whole existence been nothing more than that of a puppet?"
+                    {STORYTXT_BACKGROUND end.jpg}
+                [/part]
+                [part]
+                    story= _ "Still, despite his misgivings about being manipulated, he feels that he can learn something from the book. Orcs had only ever been half of the story — the other had been his own people being unable to accept his use of dark magic. But, if he allowed a foolish hero to take a copy of the book, would others eventually follow in his footsteps? If enough people began to view necromancy as another magical art rather than only a corrupting power, could it eventually be accepted in society?"
+                    {STORYTXT_BACKGROUND end.jpg}
+                [/part]
+                [part]
+                    story= _ "Rumors circulate of a lich who preys on scouting patrols. A hero, gathering his loyal troops to him, decides to put an end to the evil."
+                    {STORYTXT_BACKGROUND end.jpg}
+                [/part]
+            [/then]
+        [/if]
+        [if]
+            {VARIABLE_CONDITIONAL timesForever greater_than 2}
+            [then]
+                [part]
+                    story= _ "The book still sits on a dusty shelf in Mal Keshar’s keep, untended and untouched, but constantly plaguing the forefront of the lich’s mind."
+                    {STORYTXT_BACKGROUND end.jpg}
+                [/part]
+                [part]
+                    story= _ "If he allowed a foolish hero to take a copy of the book, would others eventually follow in his footsteps? Would the author’s plan actually work, or would people continue to view necromancy as only a corrupting power? One voice tells him that being accepted by society should not be of any concern to him. Another says that it doesn’t hurt to try. He does not come to any conclusions, but a copy of the tome sits there on a shelf in his lair, waiting to be taken."
```

Comment:
It's great to read your prose, it's much better than mine. I love the dwarves :D

My single reservation is over whether the hero gets a copy, or the original. It being a copy suggests that Malin has learnt how to create wards to make it indestructible, otherwise the hero could destroy it.

Reply:
It seems reasonably feasible to me that Malin could learn to create a ward that makes the book impervious to normal attacks (although one could imagine that a strong mage could still break through it). Maybe the book itself even details a way to do it? Either way, Malin is fairly powerful at this point, so that's not beyond his ability.